src/FlipTest.java

```diff
@@ -0,0 +1,34 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class FlipTest {
+    private UnoFlipModel unoFlipModel;
+    private Card testCard;
+
+    @Before
+    public void setUp(){
+        unoFlipModel = new UnoFlipModel();
+        unoFlipModel.getPlayers().add(new Player("test1"));
+        unoFlipModel.getPlayers().add(new Player("test2"));
+        unoFlipModel.setNumPlayers(2);
+        testCard = new Card(Card.Rank.FLIP, Card.Colour.RED,Card.Rank.FLIP,Card.Colour.ORANGE);
+    }
+
+    @Test
+    public void testExecuteSequence(){
+        unoFlipModel.getTurnSeqs().get(15).executeSequence(testCard); // should be index 18 according to enum
```

Comment:
even in tests, it is preferable that we use constants instead of literal 15 so that in case we ever move the enums around it'll be easier for us to know what needs to be changed (since constants will be basically acting as labels)